### ISSUE TITLE
Automatically Install libc debug symbols for i386 on Ubuntu

### ISSUE THREAD

**Issue by zachriggle** [pwndbg/pwndbg#764]
Per #635 some commands only work with libc debug symbols installed.

We currently auto-install the e.g. `libc6-dbg` package on Ubuntu, but many users may debug 32-bit apps and will lack symbols for that.

We should automatically attempt to install `libc6-dbg:i386` as well, which may require the following:

```sh
# Add architecture
sudo dpkg --add-architecture i386 

# Fetch i386 package info
apt-get update

# Add symbols
apt-get install -y libc6-dbg:i386

# Add libraries for compilation
apt-get install -y libc6-dev-i386 
```

**Comment by zachriggle** [comment]
It looks like we already attempt to install `libc6-dbg:i386` but do not do the `--add-architecture` step which is required for the package to be available.

https://github.com/pwndbg/pwndbg/blob/609284cee279de345dcb0706e11a0b56abe349f4/setup.sh#L27-L29

**Comment by zachriggle** [comment]
Does our `heap` command work for alternate architectures when running under qemu-user?  If so, we should probably add:

```
apt-get install -y 'libc6-*-cross' 'libc6-dbg-*-cross'
```

To install libc and debug symbols for all cross-architectures that are supported

**Comment by IdanBanani** [comment]
Why would someone want to install debug symbols for all possible architectures ?
apt-get install -y 'libc6-*-cross' 'libc6-dbg-*-cross' 
is it such light-weight?

**Comment by disconnect3d** [comment]
TL;DR: To make it more convenient for ppl running binaries under qemu-user emulation. It's probably not as much lightweight as installing only the _native_ arch but I don't think people  care that much as long as those packages are official ones?

**Comment by disconnect3d** [comment]
This being said, debugging via qemu-user is kinda broken for now as we don't determine memory pages well.

**Comment by gsingh93** [comment]
I think we can close this issue. @disconnect3d?

**Comment by disconnect3d** [comment]
Yeah, we have some heap heuristics thanks to @lebr0nli that should at least mitigate this issue. We can close this.
